specs/driver_spec.rb

```diff
@@ -77,4 +78,45 @@
       driver.average_rating.must_equal 0
     end
   end
+
+  describe "total_revenue method" do
+    it "returns the total revenue of driver trips" do
+      # Arrange
+
+      trip_dispatcher = RideShare::TripDispatcher.new
+      # Act
+
+      driver = trip_dispatcher.find_driver(1)
+      # Assert
+
+      driver.total_revenue.round(2).must_equal 115.97
+    end
+  end
+
+  describe "total_hours method" do
+    it "returns the total amount of hours" do
```

Comment:
Other questions that I would like to see these tests ask, for all the wave 1 methods (on both `Driver` and `Passenger`):

- What happens if the driver or passenger has no trips?
- What happens if the driver or passenger has an incomplete trip?